### DISCUSSION TITLE
Ranked Play Mania Feedback from community

### DISCUSSION THREAD

**Discussion by tailsdk** [ppy/osu#25859] [Ideas]
Since ranked play is coming out in just a few weeks I went and put in some effort to gather feedback from the mania community.

I have split them into different categories so it should be easier to keep track of which things important.

# High Priority
- Mod score multipliers
They are currently very different from stable mania and it seems most people prefer keeping them the same as on stable so for example no multiplier for DT and a 0.5x multiplier for HT

- Bonus Score
A lot of people seem to not be liking the scoring system and rather have a cap at 1 mil. A lot of this comes because it has been a way of telling how good your score is in stable and because the bonus score makes it different for every map since the score just gets added onto the 1 mil.

- Mod PP Reward
A lot of people have shown concerns around the difficulty adjust mod since it allowing you to change values can make some maps a lot easier. This is moreso a question on what the plan is around these newly added mods and PP.

# Gameplay/Skinning (for later not required, but something people want before they can get comfortable playing on lazer)
- Long Note Tails overlap the note afterwards. A lot of skins have a black box around the Long Note Tail element and it overlaps the following notes. See this skin here for example https://drive.google.com/file/d/1tPNPZ2UrHMgfiDsebtPbZBDPdjbXW1wU

- Scroll Speed does not match stable
A lot of people were saying that the scroll speeds does not match the same scroll speeds on stable and further more that they miss the more precise picking for scroll speeds. A solution could be to just input the ms value into a field instead for extra customizability

- Hit Position is not easily changeable
A lot of players change their hit position all the time and this is something that is currently really missing for gameplay.

- 10k+ skinning have some issues
Stage split 0 for example does not result in no split between the 2 stages when converting a skin over and column width does not get applied like on stable on the second playfield.

- Legacy Combo counter flashes
A lot of people on stable uses the combo counter as a place to focus on when playing and because the combo counter is not centered and it flashes when it goes up unlike in stable it makes it really hard for some people to get used to playing

**Comment by peppy** [comment]
> Bonus Score

Will eventually be shown separately at results screen, which may help with this.

> Mod score multipliers

See release video; see changelog. Multipliers will be figured out going forward.

> Mod PP Reward

Difficulty adjust isn't even giving pp. Chill.

**Reply by tailsdk** [reply]
Alright those were just the main concerns I received so good to hear that these are not issues. ty for the quick response

**Comment by Givikap120** [comment]
Why no multiplier for DT
DT is objectively harder than NM, why no multiplier

**Reply by tailsdk** [reply]
The main reason is that people like the 1 million score cap in mania since it has always been a thing. It makes it easy to quickly compare scores and people can just turn on the DT leaderboard to see DT scores which is what they did in stable. This is a pretty unanimous thought in the community
